Source/diablo.cpp

```diff
@@ -1288,7 +1288,19 @@ void PressChar(int vkey)
 		difficulties[1] = "Nightmare";
 		difficulties[2] = "Hell";
 		sprintf(pszStr, "%s, mode = %s", gszProductName, difficulties[gnDifficulty]);
-		NetSendCmdString(1 << myplr, pszStr);

```

Comment:
You could remove pszStr declaration then

Reply:
this code is only for verifying that this is a proper fix, if it works, then it will be removed later

Reply:
:+1: